test/debug/debug-deploy.sh

```diff
@@ -18,18 +18,15 @@ fi
 
 BUILT_IMAGE=$(ko build github.com/kubearchive/kubearchive/cmd/${COMPONENT%%-*} --debug)
 
-YAML=$(mktemp --suffix=.yaml -t kubearchive-XXX)
-cat ${SCRIPT_DIR}/patch-${COMPONENT}.yaml | envsubst > ${YAML}
-
 # Pause the deployment updates
 kubectl -n kubearchive rollout pause deployment kubearchive-${COMPONENT}
 
 # Set the new image
 kubectl -n kubearchive set image deployment kubearchive-${COMPONENT} ${CONTAINER}=${BUILT_IMAGE}
 # Add -- as the first argument
-kubectl -n kubearchive get deployment kubearchive-${COMPONENT} -o json | \
-jq '.spec.template.spec.containers |= map(select(."name"=="'${CONTAINER}'") .args |= ["--"] + .)' | \
-kubectl apply -f -
+#kubectl -n kubearchive get deployment kubearchive-${COMPONENT} -o json | \
+#jq '.spec.template.spec.containers |= map(select(."name"=="'${CONTAINER}'") .args |= ["--"] + .)' | \
+#kubectl apply -f -
```

Comment:
Why not just delete these lines?

Reply:
I'll remove them, I forgot to do it after I saw this works now.